treeherder/webapp/api/bugzilla.py

```diff
@@ -0,0 +1,46 @@
+import requests
+from django.conf import settings
+from rest_framework import (status,
+                            viewsets)
+from rest_framework.decorators import list_route
+from rest_framework.response import Response
+
+from treeherder.etl.common import make_request
+
+
+class BugzillaViewSet(viewsets.ViewSet):
+    # permission_classes = (IsAuthenticatedOrReadOnly,)
+
+    @list_route(methods=['post'])
+    def create_bug(self, request):
+        """
+        Create a bugzilla bug with passed params
+        """
+
+        params = request.data
+        url = settings.BZ_API_URL + "/rest/bug"
+        headers = {
+            'X_BUGZILLA_API_KEY': settings.BZ_API_KEY
+        }
+        data = {
+            'product': params["product"],
+            'component': params["component"],
+            'summary': params["summary"],
+            'keywords': 'intermittent-failure',
```

Comment:
Now that I'm thinking about it, I should probably not hardcode the intermittent keyword into this so it can be reused for other things.